ADD MODULE: module media file | module cli

fahrenheit451fra opened the pull request:
I just added two modules like [@hrsetyono](https://github.com/hrsetyono) : 
A module for CLI and one for image transfer

Thank you @jsongerber for maintaining this plugin ❤️

fahrenheit451fra committed:
ADD MODULE: module media file | module cli

jsongerber commented:
Thank you for the pull request and the kind words.
That's quite a big change, I will need some time to check that it doesn't break anything, but I will merge if everything is ok

bdlowery commented:
i dont trust it  👀

I agree, thank you for maintaining the only fork of the super mega expensive wp migrate pro 🙏

jsongerber commented:
Hi, just passing by to say I have not forgotten this pull request.

But there is two problems, that I need to find the time to fix:
- The readme are in three different places, I think that make no sense and they should be merged
- I installed the version in the pull request in a WP installation where I had `wp-sync-db `and `wp-sync-db-media-files`, that triggered a non recoverable 500 error, that's not acceptable and should not break a working WP installation

I don't yet how I will handle this on the GitHub side, I'll maybe close this pull request and open multiple ones, I'll mention you as a contributor if I do that.

jsongerber committed:
integrate media files

jsongerber committed:
clean + version

jsongerber committed:
cli module wip

jsongerber committed:
fix cli things

jsongerber committed:
remove unused test

jsongerber committed:
change vendor to lib to prevent ignores

jsongerber committed:
generate artifact on new release to include composer autoload

workflow_dispatch

now the start of the workflow commits

workflow from feature branch?

fix

jsongerber committed:
add the tag to the artifact name

jsongerber committed:
readme wip

jsongerber committed:
add profiles cli command

jsongerber committed:
add usage and wp cli usage

jsongerber committed:
add message if no autoload

jsongerber committed:
profile command format

jsongerber committed:
readme

jsongerber committed:
better error message if installed from wrong zip

jsongerber commented:
I changed some class loading to use `PSR-4` and I still have problem installing with the GIT Updater plugin so I opened [an issue](https://github.com/afragen/git-updater/issues/1104) to try to fix this, but If you download the plugin from the [latest release](https://github.com/jsongerber/wp-sync-db/releases/latest) it should work
